mep/common/utils.py

```diff
@@ -91,7 +91,11 @@ def alpha_pagelabels(paginator, objects, attr_meth):
             abbr += char   # or abbr = label[:j]
             # if abbreviation is different from neighboring labels
             # at the current length, then use it
-            if abbr not in (next_label[:j], prev_label[:j]):
+            # - compare case-insensitively
+            # NOTE: should possibly ignore trailing punctuation here...
+            # possible to get "Gray" / "Gray,"
```

Comment:
glad you flagged this - I noticed a bunch of "Gray," type stuff last time I was playing around with the page labels and I think it looks weird. If we can avoid showing it in the page labels I think that would be awesome.

Reply:
I'm not quite sure how to implement the solution yet... Might be as simple as stripping trailing whitespace and punctuation when comparing; then "Gray" and "Gray, " would match until the next letter after the comma is included. Should we make an issue to document the problem?

Reply:
created #463